vendor/cookbooks/rails/templates/default/app_passenger_nginx.conf.erb

```diff
@@ -25,6 +25,9 @@ server {
 
   passenger_enabled on;
   passenger_app_env <%= @rails_env %>;
+
+  client_max_body_size <%= @client_max_body_size %>;
```

Comment:
What if the variable is not passed in? eg `nil`, then it will break the nginx config right?